devicetypes/smartthings/zwave-switch-secure.src/zwave-switch-secure.groovy

```diff
@@ -22,7 +22,8 @@ metadata {
 
 		fingerprint inClusters: "0x25,0x98"
 		fingerprint deviceId: "0x10", inClusters: "0x98"
-		fingerprint mfr: "0086", prod: "0003", model: "008B", deviceJoinName: "Aeon Labs Nano Switch"
+		fingerprint mfr: "0152", prod: "0003", model: "0002", deviceJoinName: "iTec Home Light Switch"
+        fingerprint mfr: "0086", prod: "0003", model: "008B", deviceJoinName: "Aeon Labs Nano Switch"
```

Comment:
@iTecHome please use tabs as indentation instead of spaces

Reply:
apologies for the delay in responding. I think we have updated it successfully now but would be grateful if you would confirm as it's the first time we have published a request and I am still finding my way.

Reply:
looks like you've still got tabs instead of spaces on this line, and we've updated our join names so you have conflicts now that you'll need to resolve.

Reply:
@greens I presume you meant we have still got spaces instead of tabs? in which case it should be ok now. Also updated the join name according to the following 'The pairing instructions should be named in the format <Brand Name> <ZigBee /Z Wave> <Unique Family Name> (e.g. Samsung ZigBee EnergyMaster)' mentioned in the guidance notes. If there is a newer version please advise.

Reply:
There's not a newer version published, but we did go through and change all of our join names, so your changes conflict with the master version now.

Reply:
apologies , I think I am missing something. I changed the spaces to tabs and changed the device join name back to "iTec Home Z-Wave Light Switch" as requested, in the device handle publication request (not in github) and re-submitted it. Should I have done this somewhere else?

Reply:
I think at this point you'll have to either do a rebase in github or re-submit (and link new PR here). Our submission process doesn't keep changed files up-to-date with the master versions in our repository. So if you look at the bottom of this PR, you'll see:

> Conflicting files
>`devicetypes/smartthings/zwave-switch-secure.src/zwave-switch-secure.groovy`

because the version of the targeted file has changed on the master branch since this PR was opened.

Reply:
new PR is 

[https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/32288]

Reply:
Ok, let's try again! It seems the master files used to create device handlers in smarthings developers area are not synced with the master on git so we were starting from outdated info. New pull #33592 created using master from git and ONLY adding iTec join name so should be correct.

It would be good if the masters in the smartthigns developers area were updated to save us all some time.